schedule/yast/encryption/crypt_home.yaml

```diff
@@ -0,0 +1,29 @@
+---
+description: >
+  Conduct installation with encrypted home partition.
+name: crypt_home
+schedule:
+  - installation/bootloader_start
+  - installation/welcome
+  - installation/accept_license
+  - installation/scc_registration
+  - installation/addon_products_sle
+  - installation/system_role
+  - installation/partitioning
+  - installation/partitioning/separate_home
+  - installation/partitioning_finish
+  - installation/installer_timezone
+  - installation/user_settings
+  - installation/user_settings_root
+  - installation/installation_overview
+  - installation/disable_grub_timeout
+  - installation/start_install
+  - installation/await_install
+  - installation/logs_from_installation_system
+  - installation/reboot_after_installation
+  - installation/grub_test
+  - installation/decrypt_disks
+  - installation/first_boot
+  - console/validate_encrypt
```

Comment:
[verify_separate_home](https://openqa.suse.de/tests/4723109#step/verify_separate_home/1) should be another module to add here or explain why doesn't work and evaluate how much would be to adapt it.

Reply:
i think it could be used. i did not try. the task was to make encryption for home and validate this. And i think the test_data and the `console/validate_encrypt` validate the separate partition anyway.

Reply:
does it? `console/validate_encrypt` seems to understand only about '/dev/vda3'. How can this module ensure that vda3 is the one with mount point in /home?

Reply:
first of all the test_data are hardcoded. this ensures that you are working with the correct partition. But to be honest it is not hard to add the verify_separate_home. i can do it. i did not for the reasons above. do you think i should after the discussion?

Reply:
correct me if I'm wrong, but do we select /home explicititly [here](http://aquarius.suse.cz/tests/3344#step/separate_home/16) if yes, we are fine, if not, is there any other point where we can ensure /home is the mount point for /dev/vda3?

Reply:
correct. there are a few. one is the one you linked. Another is http://aquarius.suse.cz/tests/3344#step/validate_encrypt/11 or http://aquarius.suse.cz/tests/3344#step/validate_encrypt/14. Because of the hardcoded data the '/dev/vda3' should always point to the home

Reply:
if dev mapper does its job well yes, cr_home is directly related with vda3, but still I don't see that we ensure the mount point without that module, we just trust that the dev mapper does its job giving it that name, if those names were switched for some bugs...(it is a remote possibility though). When we test the system we should be very explicit and not trust in the intermediate steps, that is my point.